src/api.js

```diff
@@ -433,3 +433,50 @@ api.declare({
   }
   return res.status(404).send();
 });
+
+api.declare({
+  name: 'createStatus',
+  title: 'Post a status against a given changeset',
+  description: [
+    'For a given changeset (SHA) of a repository, this will attach a "commit status"',
+    'on github. These statuses are links displayed next to each revision.',
+    'The status is either OK (green check) or FAILURE (red cross), ',
+    'made of a custom title and link.',
+  ].join('\n'),
+  stability: 'experimental',
+  method: 'post',
+  // route and query matches github API
+  // https://developer.github.com/v3/repos/statuses/#create-a-status
+  route: '/repository/:owner/:repo/statuses/:sha',
+  input: 'create-status.json',
+  scopes: [['github:create-status:<owner>/<repo>/<sha>']],
```

Comment:
We don't really want scopes based on sha, right? Branch makes a bit more sense in this context. That way we can control whether or not you can set scopes on master, but a sha-based scopes is only useful once.